src/i18n/video/video.fr.json

```diff
@@ -0,0 +1,12 @@
+{
+  "duration": {
+    "to": "à",
+    "not-specified": "Durée non spécifiée"
+  },
+  "error": {
+    "only-youtube-and-insta": "Seules les vidéos YouTube et Instagram peuvent être affichées !",
+    "invalid-url": "URL vidéo non valide !",
+    "invalid-youtube": "L'URL YouTube fournie ne peut pas être analysée ! La vidéo ne peut pas être affichée."
```

Comment:
```diff
- … !
+ …!
```

Reply:
Thought the same until DeepL repeatedly put a space before exclamation marks. After a quick internet search it turns out this is actually a quirk of the French language and not a mistake.

Reply:
just confirmed with a french friend of mine :)